### DISCUSSION TITLE
Giudizio Sospeso

### DISCUSSION THREAD

**Discussion by luigirotondi2009** [iisgiua/giuaschool#718] [Richieste di aiuto] [answered]
In caso di giudizio sospeso, il voto della prova scritta per l'ammissione alla classe successiva o al contrario, dove si pubblica?

**Comment by trinko** [comment, marked as answer]
Attualmente non è previsto l'inserimento nel registro di questa valutazione. Si procede direttamente a registrare il voto stabilito dal Consiglio di Classe nello scrutinio.

E' possibile che riesca a modificare questa procedura per la prima settimana di luglio, permettendo l'inserimento nelle proposte di voto.
